pytest_playwright/pytest_playwright.py

```diff
@@ -48,6 +55,20 @@ def pytest_configure(config: Any) -> None:
     )
 
 
+# Making test result information available in fixtures
+# https://docs.pytest.org/en/latest/example/simple.html#making-test-result-information-available-in-fixtures
+@pytest.hookimpl(tryfirst=True, hookwrapper=True)
+def pytest_runtest_makereport(item: Any) -> Generator[None, Any, None]:
+    # execute all other hooks to obtain the report object
+    outcome = yield
+    rep = outcome.get_result()
+
+    # set a report attribute for each phase of a call, which can
+    # be "setup", "call", "teardown"
+
+    setattr(item, "rep_" + rep.when, rep)
```

Comment:
Why `rep` or `rep_setup` and not `result`? After all, it is `get_result()` in the pytest api?

Reply:
See the comment above the method definition. This is fully taken from their docs: https://docs.pytest.org/en/latest/example/simple.html#making-test-result-information-available-in-fixtures